### ISSUE TITLE
Latest theplayer with android sdk 7.0.0 is failing

### ISSUE THREAD

**Issue by aurimasmi** [THEOplayer/react-native-theoplayer#304]
Hoe to reproduce:

1. Update react-native-theoplayer to: v3.10.3
2. Make sure THEOplayer_sdk is 7.0.0
3. Observe
```
react-native-theoplayer/android/src/main/java/com/theoplayer/PlayerConfigAdapter.kt: (8, 39): Unresolved reference: AdsConfiguration
react-native-theoplayer/android/src/main/java/com/theoplayer/PlayerConfigAdapter.kt: (9, 39): Unresolved reference: AdPreloadType
```

**Comment by tvanlaerhoven** [comment]
Hey @aurimasmi , indeed we are releasing a new react-native-theoplayer in the next day(s) with 7.0 support. It is currently being tested on this branch:
https://github.com/THEOplayer/react-native-theoplayer/pull/302

**Comment by ceyhun-o** [comment]
Hi @aurimasmi,

We released RN SDK 7.0.0, which makes it compatible with THEOplayer 7. On the Android side, I recommend using the latest (7.1.0) as there are some important fixes on this version.

Did you have the chance to give it a try? Can you build without errors now?

Kind regards,
Ceyhun

**Comment by tvanlaerhoven** [comment]
@aurimasmi closing this ticket for now, let us know if you need additional support.
